src/main/java/org/osc/controller/nsc/api/SampleSdnRedirectionApi.java

```diff
@@ -72,8 +73,8 @@ public InspectionHookElement getInspectionHook(NetworkElement inspectedPort, Ins
         try {
             return this.utils.findInspHookByInspectedAndPort(inspectedPort, inspectionPort);
         } catch (Exception e) {
-            LOG.error(String.format("Exception finding Network Element (Inspected %s ; Inspection Port %s):",
-                    "" + inspectedPort, "" + inspectionPort), e);
+            LOG.error("Exception finding Network Element (Inspected %s ; Inspection Port %s):", "" + inspectedPort,
```

Comment:
%s should be replaced  by {}. see previous comment. also have string concatenation with "" + inspectedPort which is not needed

`LOG.warn("Attempt to delete network element for id {} and network element not found, no-op.", id);
`

Reply:
Double quote was already part of earlier code, will clean this up as proposed.